torch/_dynamo/variables/builder.py

```diff
@@ -443,9 +444,10 @@ def index_source(key):
                     return key
 
             result = {
-                k: VariableBuilder(
-                    self.tx, GetItemSource(self.get_source(), index_source(k))
-                )(value[k]).add_guards(guards)
+                k: LazyVariableTracker.create(
+                    value[k],
+                    source=GetItemSource(self.get_source(), index_source(k)),
+                ).add_guards(guards)
```

Comment:
How come this is used in particular for values of dictionaries?

Reply:
The next PR in the stack turns it on in more places.  Dict keys seem like something more likely to be unused.